discovery-provider/src/models/track_route.py

```diff
@@ -0,0 +1,37 @@
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    Boolean,
+    PrimaryKeyConstraint,
+)
+from .models import Base
+
+
+class TrackRoute(Base):
+    __tablename__ = "track_routes"
+
+    # Actual URL slug for the track, includes collision_id
+    slug = Column(String, nullable=False)
+    # Just the title piece of the slug for the track, excludes collision_id
+    # Used for finding max collision_id needed for duplicate title_slugs
+    title_slug = Column(String, nullable=False)
+    collision_id = Column(Integer, nullable=False)
+    owner_id = Column(Integer, nullable=False)
+    track_id = Column(Integer, nullable=False)
+    is_current = Column(Boolean, nullable=False)
+    blockhash = Column(String, nullable=False)
```

Comment:
let's include the txhash here as well - it gives us more granularity in the data model in case we need to monitor later on

Reply:
checkout User/Track or one of the other data objects in models.py